### DISCUSSION TITLE
About 'My CRIPT'

### DISCUSSION THREAD

**Discussion by c-heat16** [C-Accel-CRIPT/Python-SDK#388] [Development Discussions]
Few comments/questions regarding the 'My CRIPT' tab of [criptapp.org](https://criptapp.org/):

1. What is the expected behavior when deleting a project in 'My CRIPT'? A message is presented saying that all child nodes will be deleted, which I take to mean all associated collections/inventories/materials/etc., but that doesn't seem to be the case. After deleting a project in 'My CRIPT', the associated collections/inventories are still present in the 'BROWSE' tab.
2. I think it would be nice if 'My CRIPT' would present all nodes associated with your account, instead of only Project nodes which it seems to be doing now. When collections/inventories exist, but were not deleted along with the associated project (above), one must browse through all 150+ collections present in all of CRIPT.
3. One such inventory is 'My Test Pyrolysis Inventory 2' with corresponding URL [https://app.criptapp.org/Inventory/6963009e-9179-4647-90cf-45ea91311a6d](https://app.criptapp.org/Inventory/6963009e-9179-4647-90cf-45ea91311a6d). This inventory appears under [https://app.criptapp.org/inventory/](https://app.criptapp.org/inventory/) (although is private), but clicking on the link yields a completely bank page. I have tried this on two browsers, so I don't think it is a browser issue. I can try to delete this using [`api.delete()`](https://c-accel-cript.github.io/Python-SDK/api/api/#cript.api.api.API.delete), but left it as-is in case it is an interesting debugging artifact.

**Comment by nh916** [comment]
Hi @c-heat16, thank you so much for asking. I think these might be frontend issues. I'm going to tag our frontend software engineer @bearmit, and if there is anything that I can help with please feel free to tag me.
